common/scripted_effects/wc_unit_scripted_effects.txt

```diff
@@ -147,4 +147,51 @@ spawn_death_knight_troops_effect = {
 		maintenance = no
 		earmark = death_knights_forces
 	}
+}
+spawn_cthun_guard_troops_effect = {
+	spawn_unit = {
+		owner = ROOT
+		province = PREV
+		home = PREV
+		earmark = old_god_guard
+		troops = {
+			pikemen = { 480 480 }
+			knights = { 240 240 }
+			swarm = { 3600 3600 }
+		}
+		reinforces = yes
```

Comment:
`recruit_old_god_guard` has trigger `NOT = { has_earmarked_regiments = old_god_guard }`. Thus, if they reinforce, most likely you won't be able to use the decision more than once.
It becomes especially sad when you get the next level of the Old God.

Reply:
they get disbanded on peace

Reply:
> they get disbanded on peace

@sejtalk can't confirm.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79079089-0388a880-7d1e-11ea-9abc-3111933b3e03.png)
![image](https://user-images.githubusercontent.com/46384992/79079099-10a59780-7d1e-11ea-8f93-4994efe94c6a.png)


</details>